src/core/hle/service/nvdrv/devices/nvhost_vic.h

```diff
@@ -87,15 +169,36 @@ class nvhost_vic final : public nvdevice {
     };
     static_assert(sizeof(IoctlUnmapBufferEx) == 0x3C, "IoctlUnmapBufferEx is incorrect size");
 
+    struct IoctlSubmitStub {
```

Comment:
```suggestion
    struct IoctlSubmit {
```

It's better to document what needs to be done to implement this, rather than naming it as a stub. This way when the proper implementation is done, no renaming needs to take place.